tests/Commands/MakeEnumTest.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+namespace Spatie\Enum\Laravel\Tests\Commands;
+
+use Spatie\Enum\Laravel\Tests\TestCase;
+
+class MakeEnumTest extends TestCase
+{
+    const WEEKDAY_PATH = __DIR__.'/../../vendor/orchestra/testbench-core/laravel/app/Enums/WeekDay.php';
+
+    protected $content;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        @unlink(self::WEEKDAY_PATH);
+        $this->content = null;
+    }
+
+    /** @test */
+    public function it_can_run_the_make_command(): void
+    {
+        $this->runMakeCommand();
+    }
+
+    /** @test */
+    public function it_can_make_an_enum_with_methods(): void
+    {
+        $methods = [
+            'monday',
+            'tuesday',
+            'wednesday',
+            'thursday',
+            'friday',
+            'saturday',
+            'sunday',
+        ];
+
+        $this->runMakeCommand([
+            '--method' => $methods,
+        ]);
+
+        foreach($methods as $method) {
+            $this->assertMethodDocTag($method);
+        }
+    }
+
+    /** @test */
+    public function it_can_make_an_enum_with_methods_studly(): void
+    {
+        $values = [
+            'Monday' => 'monday',
```

Comment:
How would this command be invoked in real situations? Wouldn't it be quite a hassle to write all values on the commandline?

Reply:
I would say that it's something like the make commands for models, migrations and so on. They also have a lot of options to configure the resulting model but most times you simply use `make:model Post` and do the rest in the created file itself.
I would keep them in to allow custom extensions like `make:enum:by-model` which reads values from DB and puts them in place or whatever.
The other solution would be a JSON file - but I don't think that someone wants to create a JSON file to call a make command?

Reply:
Good enough. I do think we need to be a little careful not to "overbloat" this package from the start. It's easy to add new features, but it's a lot harder to maintain remove them afterwards.

But I do think this make command is a valuable one to have, so let's go ahead and merge it 👍